src/midi.js

```diff
@@ -3,7 +3,14 @@
 const fs = require('fs');
 const assert = require('assert');
 const jsmidgen = require('jsmidgen');
-
+var transposition = 0;
+/**
+ * Takes an integer and transposes all notes to a different middle C octave.
+ * @param {Integer} octaveIndex		The new octave for middle C.
+ */
+function setMiddle(octaveIndex){
```

Comment:
Any non Numerical value. The objective is to cover edge cases. Knowingly or unknowingly, someone may call this method something like `scribble.setMiddleC('c3')` 

There are folks who script stuff that decides root notes, octaves, patterns etc randomly. For some reason that script ends up calling scribble.setMiddleC with something completely unexpected value, like `scribble.setMiddleC(true)` OR  `scribble.setMiddleC('D#')` then our code must let them know that by throwing an error. Hence the test case to cover invalid values.